test/verify/main.cpp

```diff
@@ -47,5 +47,6 @@ int main(int argc, const char* argv[])
 {
     run_verify rv;
     rv.add_validation_for("gpu", &validate_gpu);
+    rv.disable_test_for("cpu", {""});
```

Comment:
It would be better to do `{}` instead. Of course, we still wont get full code coverage, but I think its fine.